stdlib/src/collections/inline_list.mojo

```diff
@@ -20,6 +20,7 @@ from collections import InlineList
 """
 
 from sys.intrinsics import _type_is_eq
+from memory.unsafe import UnsafeMaybeUninitialized
```

Comment:
**Suggestion** Shouldn't this import be `from memory.maybe_uninitialized import UnsafeMaybeUninitialized`?  The type isn't in `memory.unsafe` and we don't hard error on missing imports.

Reply:
I just fixed this up internally FYI.

Reply:
Yeah, imports are still a bit funky